src/app/(admin)/(auth)/layout.tsx

```diff
@@ -0,0 +1,22 @@
+import { Inter } from 'next/font/google'
+
+import AuthCarousel from './_components/AuthCarousel'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode
+}>) {
+  return (
+    <div className={`h-screen w-screen ${inter.className}`}>
+      <div className='flex h-full w-full justify-between'>
+        {children}
+        <div className='right relative h-full w-full flex-[1] bg-[#EFEAE0]'>
```

Comment:
what is this class "right" for? Also instead of using hex codes here for colors, we can define these in tailwind config and use keyword here.

Reply:
class is for convenience and since the hex code is only used here that's why i hardcoded it here